backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -153,7 +155,48 @@ private Specification<TestEvent> buildTestEventSearchFilter(
       return p;
     };
   }
+  // methods to replace the methods that return List<TestEvent> once we get
+  // backwards compability issues resolved
+  @Transactional(readOnly = true)
+  @AuthorizationConfiguration.RequirePermissionReadResultListAtFacility
+  public Page<TestEvent> getTestEventsResultsPage(
+      UUID facilityId,
+      UUID patientId,
+      TestResult result,
+      PersonRole role,
+      Date startDate,
+      Date endDate,
+      int pageOffset,
+      int pageSize) {
+
+    PageRequest pageRequest =
+        PageRequest.of(pageOffset, pageSize, Sort.by("createdAt").descending());
+
+    return _terepo.findAll(
+        buildTestEventSearchFilter(facilityId, patientId, result, role, startDate, endDate),
+        pageRequest);
+  }
 
+  @Transactional(readOnly = true)
+  @AuthorizationConfiguration.RequirePermissionViewAllFacilityResults
+  public Page<TestEvent> getAllFacilityTestEventsResultsPage(
```

Comment:
nit: I would suggest changing the name of this method to `getOrganizationTestEventsResultsPage`, it's easier to parse at a glance

Reply:
Definitely think renaming for AllFacility --> Organization makes sense! Was just thinking of using the old method name convention once we clean up the backend, since I didn't think calling out the type in the method name is super necessary. 
 
Not sure if you have a preference for `getOrganizationTestEventsResultsPage` vs just `getOrganizationTestEventsResults` but will probably go with the later if you're ambivalent.

Reply:
The latter works for me! I was mostly concerned with the AllFacility --> Organization pattern, but I'm all for dropping the `Page` once we've consolidated the methods a bit 😄